docs/labwc-theme.5.scd

```diff
@@ -195,6 +195,52 @@ elements are not listed here, but are supported.
 	Height of boxes in workspace switcher in pixels. Setting to 0 disables
 	boxes. Default is 20.
 
+*snapping-preview.region.fill* [yes|no|auto]
+	Show filled rectangle as preview when window is snapped to a region.
+	Otherwise, outlined rectangle is shown instead.
+	When *auto* is specified, the style of a rectangle is automatically
+	determined depending on the current rendering backend which can be specified
+	with *WLR_RENDERER* environment variable.
+	Default is *auto*.
+
+*snapping-preview.edge.fill* [yes|no|auto]
+	Show filled rectangle as preview when window is snapped to an edge.
+	See *snapping-preview.region.fill* above for details.
+	Default is *auto*.
+
+*snapping-preview.region.bg.color*
+	Color of filled rectangle shown as preview when window is snapped to a
+	region.
+	Default is #8080b380.
+
+*snapping-preview.edge.bg.color*
+	Color of filled rectangle shown as preview when window is snapped to an edge.
+	Default is #8080b380.
+
+*snapping-preview.region.border.width*
+	Border width of outlined rectangle shown as preview when window is snapped to
+	a region.
+	Default is 1.
+
+*snapping-preview.edge.border.width*
+	Border width of outlined rectangle shown as preview when window is snapped to
+	an edge.
+	Default is 1.
+
+*snapping-preview.region.border.color*
+	Color(s) of outlined rectangle shown as preview when window is snapped to a
+	region.
+	Possible value is a color or up to 3 colors separated by a comma
```

Comment:
Similar thing here, I think the line break is not necessary to split the description and the value definition. Same for `snapping-preview.edge.border.color`.